custom_tvm_ops/tests/layer_norm_test.py

```diff
@@ -0,0 +1,206 @@
+import numpy
```

Comment:
this file should all live within `test/test_operators.py` or a new file `test/test_custom_operators.py`

Reply:
So this goes in the same spirit as that of the other comment. My point was to keep custom tvm ops implemented in a separate place and tested independently. They do not have any dependency on torch_tvm.